mayastor-test/test_replica.js

```diff
@@ -60,6 +60,7 @@ function destroyTestDisk(done) {
 }
 
 function createGrpcClient(service) {
+  console.log('Starting client at ', endpoint);
```

Comment:
I suppose this was used for debugging and can be removed now

Reply:
I took out a lot of my debugging prints but not this, as I thought it was not unreasonable for the test to announce the socket for the interface the test suite was about to test.

Reply:
Particularly as some of the tests now run twice, with different sockets.

Reply:
In general there should be no output from mocha test suite except the green check marks (or red if it fails). That said sometimes the code we are testing logs messages to stderr - nothing we can do about it. And perhaps there are other few exceptions like printing log output of mayastor process or csi agent process (as there is no way to get this output otherwise). But log messages like "Starting something ... ", "Doing this or that" should be avoided as they clutter the output more than necessary. In the long term we won't run the same tests against two endpoints, but there will be just one endpoint. Then how useful is the information that I'm running the tests against some particular endpoint. But we can remove that later when the obsolete code in csi-agent goes away 👍

Reply:
I can see the reason you are describing, but when I run the tests I get thousands of lines of output, as it dumps out the std{out,err} of the processes it was running.  If all you got was the grey text with the ticks, I would have hesitated to add this kind of thing.  So OK, I'll remove it again when I get rid of the agent's gRPC and the tests go back to a single socket.